WcaOnRails/app/helpers/application_helper.rb

```diff
@@ -52,8 +52,12 @@ def wca_excerpt(html, phrase)
     wca_highlight(excerpted, phrase)
   end
 
-  def wca_highlight(html, phrases)
-    text = ActiveSupport::Inflector.transliterate(strip_tags(html)) # TODO https://github.com/cubing/worldcubeassociation.org/issues/238
+  def wca_highlight(html, phrases, accept_special)
```

Comment:
Lol, nice fix, but doesn't adding an extra argument to this method break other people using it?